_posts/2025-04-02-debezium-3-1-final-released.adoc

```diff
@@ -0,0 +1,999 @@
+To address this concern, we have introduced a new configuration property, `pubsub.region` (https://issues.redhat.com/browse/DBZ-8735[DBZ-8735]).
+
+The new `pubsub.region` property allows specifying the Google Cloud region to connect, i.e. `us-central1` or `asia-northeast1`.
+When specified, Debezium will use the location-specific endpoint for PubSub in the format `<region>-pubsub.googleapis.com:443`.
+This permits connecting to the location-specific endpoint instead of the global endpoint.
+
+[NOTE]
+====
+The `pubsub.region` and `pubsub.address` configuration properties are mutually exclusive.
+If you provide a `pubsub.address` in the configuration, the `pubsub.region` will be ignored.
+====
+
+==== RabbitMQ sink supports key based routing
+
+We have changed how you can route events using configuration.
+This new approach uses a strategy-based design, that retains old behaviors and introduces the new key-based routing mechanism (https://issues.redhat.com/browse/DBZ-8752[DBZ-8752]).
+
+First and foremost, the `rabbitmq.routingKeyFromTopicName` is deprecated and will be removed in a future release.
+This functionality has been folded into the new `rabbitmq.routingKey.source` configuration property, and it can be set one to one of the following values:
+
+`static`::
+When using the _static_ routing source, the RabbitMQ sink will use the `rabbitmq.routingKey` static value you have specified in the sink's configuration.
+As this value is set in the configuration and read only during the sink startup, the value is static and does not change over the runtime of the sink.
+
+`topic`::
+When using the _topic_ routing source, the RabbitMQ sink will source the routing key based on the destination topic name.
+This mode replaces the old `rabbitmq.routingKeyFromTopicName` configuration property behavior, which is now deprecated.
+
+`key`::
+When using the **new** _key_ routing source, the RabbitMQ sink will source the routing key based on the event's record key.
+This provides the flexibility to control the routing mechanism for RabbitMQ to use the raw Debezium change event's key or by using a custom transformation to change the event's key in-flight before sending the event to RabbitMQ.
+
+
+[id=new-features-and-improvements-platform]
+=== Debezium  Platform
+
+==== What is Debezium Platform?
+
+A year ago we began this incredible journey to create a new, modern user interface for Debezium Server that aimed to ease the deployment of Debezium on Kubernetes.
+We are excited that Debezium 3.1 is the first official release of this years-long effort.
+
+The new Debezium Platform provides a modern pipeline-based approach to designing source and sink configurations, transformation chains, and more within seconds.
+You can install the Debezium Platform using `helm` as follows:
+
+[source,bash]
+----
+helm install debezium-platform --set domain.url=<your-domain> --version 3.1.0-beta1 oci://quay.io/debezium-charts/debezium-platform
```

Comment:
```suggestion
helm install debezium-platform --set domain.url=<your-domain> --version 3.1.0-final oci://quay.io/debezium-charts/debezium-platform
```

Reply:
Great catch @mfvitale appreciate that :tada: